javalin/src/main/java/io/javalin/http/util/RateLimitUtil.kt

```diff
@@ -38,6 +40,14 @@ class RateLimiter(val timeUnit: TimeUnit) {
             }
         }
     }
+
+    fun getCounter(ctx: Context) : Int? {
+       return keyToRequestCount.get(RateLimitUtil.keyFunction(ctx))
+    }
+
+    fun getRemainder(timeUnit: TimeUnit) : Long? {
```

Comment:
```suggestion
    fun remainder(timeUnit: TimeUnit) : Long? {
```

Reply:
Wait, why does it take a TimeUnit ?

Reply:
You will want a different time unit for the remainder.  e.g. take a rate limit of 5 requests per SECOND where the remaining time before the SECOND expires is in MILLISECONDS...

Reply:
Perhaps I could use a float... 0.25 left of your TimeUnit; then you would not need another TimeUnit